### ISSUE TITLE
Configure NLog FileTarget to use KeepFileOpen=true

### ISSUE THREAD

**Issue by snakefoot** [jakdaniel/NlogSerilogDotnetBenchmark#1]
Please change this location:
https://github.com/jakdaniel/NlogSerilogDotnetBenchmark/blob/b1e501e031a9f4a7164fccc3c08e797ec96cbbb8/NlogExemple/Program.cs#L58-L62
And this location:
https://github.com/jakdaniel/NlogSerilogDotnetBenchmark/blob/b1e501e031a9f4a7164fccc3c08e797ec96cbbb8/NlogExemple/Program.cs#L44-L48

To this:
```c#
 var fileTarget = new FileTarget(fileName) 
 { 
     FileName = fileName, 
     KeepFileOpen = true,
     Concurrentwrites = false,
     Layout = GetLayout(), 
 };
```
Notice these are the new defaults for NLog 5.0. Would love if you could also do the benchmark with [NLog.Web.AspNetCore ver. 5.0-rc1](https://www.nuget.org/packages/NLog.Web.AspNetCore/5.0.0-rc1) (Includes lots of performance optimizations when using `BeginScope`)

**Comment by jakdaniel** [comment]
Thanks for your help

**Comment by snakefoot** [comment]
Closed by #4
